### PR TITLE
Use container_name in compose file.

### LINKED ISSUE #111: Add container_name to all containers in docker-compose
so that we management of containers can be more usable

https://docs.docker.com/compose/compose-file/#container-name

### PR DESCRIPTION
**Pull request by dmabtrg**
This fixes #111 and #109.

### CODE REVIEW

**Comment by dmabtrg** [comment]
@jmcanterafonseca could you review this, please?

File: docker-compose.yml [hunk 1]
```diff
@@ -19,12 +21,14 @@ orion:
...
     image: fiware/authzforce-ce-server:release-4.4.1b
     hostname: authzforce
+    container_name: authzforce
     expose:
         - "8080"
 
 idm:
     image: bitergia/idm-keyrock:5.1.0
     hostname: idm
+    container_name: idm
```
Comment (jmcanterafonseca) [inline docker-compose.yml]: change to 'keyrock' and fix the other issue in this PR as well

Comment (dmabtrg) [inline docker-compose.yml]: I've added a new commit to fix #109 too.

**Comment by jmcanterafonseca** [comment]
LGTM, thanks

**Pull request merged**